.github/workflows/dispatch-restart-apps.yml

```
@@ -23,6 +23,11 @@ jobs:
       id-token: write
       contents: read
     steps:
+      - name: "Checkout GitHub Action"
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          ref: ${{ inputs.ref }}
```

actionlint (1.7.7)

property "ref" is not defined in object type {environment: string}

(expression)